cartridge/roles/metrics.lua

```diff
@@ -71,7 +73,7 @@ local function validate_routes(export)
     for _, v in ipairs(export) do
         v.path = remove_side_slashes(v.path)
         assert(type(v.path) == 'string', 'export.path must be string')
-        assert(handlers[v.format], 'format must be "json", "prometheus" or "health"')
+        assert(metrics_vars.handlers[v.format], 'format must be in handlers list')
```

Comment:
Maybe we should keep on default handlers list in error message? I think this one may confuse users.